pkg/convert/json_ngxtranslate.go

```diff
@@ -0,0 +1,61 @@
+package convert
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+)
+
+func FromNgxTranslate(b []byte) (messages model.Messages, err error) {
+	var dst map[string]interface{}
+
+	if err = json.Unmarshal(b, &dst); err != nil {
+		return messages, fmt.Errorf("unmarshal from ngx-translate to model.Messages: %w", err)
+	}
+
+	var traverseMap func(key string, value interface{}) error
+
+	traverseMap = func(key string, value interface{}) (err error) {
+		switch v := value.(type) {
+		default:
+			return errors.New("unsupported value type")
```

Comment:
This error might be better with some added details: `fmt.Errorf("usupported value type %T for key %s", value, key)`

Reply:
Thank you ;)